docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,59 @@
+# Service_Water_Heating - Rule 11-1
+**Schema Version:** 0.0.38  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-1
+
+**Rule Description:** "Where a complete service water-heating system exists, the proposed design shall reflect the actual system type. Where a service water-heating system has been designed the service waterheating type shall be consistent with design documents."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** P-RMD each SHW type
```

Comment:
Should this have a different context? Maybe P-RMD each ServiceWaterHeatingEquipment. I am not following why it would be evaluated for each type as opposed to system.

Reply:
I am struggling with the same thing

Reply:
Can SWH systems serve more than one SWH building area type? Is that accounted for already?

Reply:
I think it could certainly be the case that a SHW system could serve more than one building area type in the proposed design.